browser-checklist.md

```diff
@@ -0,0 +1,26 @@
+## Browsers
+
+- [ ] Chrome
+- [ ] Safari
+- [ ] FF
+- [ ] IE10
+- [ ] IE9
+- [ ] IE8
+- [ ] IE7
+- [ ] Opera
+- [ ] iOS8
+- [ ] Android 5
```

Comment:
It's probably worth testing both? Though to be honest, I just use the Chrome device emulation for this... is anyone testing on a non-Chrome Android browser? If so, how?